dashboard/src/app/projects/create-project/create-project.html

```diff
@@ -140,23 +140,39 @@
     <!-- Select stack -->
     <che-label-container che-label-name="{{createProjectCtrl.workspaces.length ? 'Select Workspace' : 'Select Stack'}}"
                          che-label-description="{{createProjectCtrl.workspaces.length ? 'Select existing workspace or create it from stack' : 'Select stack'}}, recipe or image used to define your environment runtime. Workspace environments are used to build and run your project.">
-      <md-radio-group ng-show="createProjectCtrl.workspaces.length" ng-model="createProjectCtrl.workspaceResource">
-        <md-radio-button value="existing-workspace">Select existing workspace</md-radio-button>
-        <div ng-if="createProjectCtrl.workspaceResource === 'existing-workspace'">
-          <create-project-workspaces che-workspace="createProjectCtrl.workspaceSelected"
-                                     che-workspace-change="createProjectCtrl.onWorkspaceChange()"/>
-        </div>
-        <md-radio-button value="from-stack">Create new workspace from stack</md-radio-button>
-      </md-radio-group>
-      <che-workspace-select-stack id="create-project-source-stack"
-                                  che-tab-name="tabName"
-                                  che-on-tab-change="createProjectCtrl.setStackTab(tabName)"
-                                  che-stack="stack"
-                                  che-stack-change="createProjectCtrl.cheStackLibrarySelecter(stack)"
-                                  che-recipe-script="createProjectCtrl.recipeScript"
-                                  che-recipe-url="createProjectCtrl.recipeUrl"
-                                  che-recipe-format="createProjectCtrl.recipeFormat"
-                                  ng-if="!createProjectCtrl.workspaces.length || createProjectCtrl.workspaceResource === 'from-stack'"/>
+      <ng-form name="workspaceResourceForm">
+        <md-radio-group ng-model="createProjectCtrl.workspaceResource"
+                        ng-init="createProjectCtrl.setWorkspaceResourceForm(workspaceResourceForm)"
+                        ng-change="createProjectCtrl.workspaceResourceOnChange()">
+
+          <!-- radio-button: existing workspace-->
+          <md-radio-button value="existing-workspace" ng-disabled="createProjectCtrl.workspaces.length === 0">Select existing workspace</md-radio-button>
+          <div ng-if="createProjectCtrl.workspaceResource === 'existing-workspace'">
+            <create-project-workspaces che-workspace="createProjectCtrl.workspaceSelected"
+                                       che-workspace-change="createProjectCtrl.onWorkspaceChange()"/>
```

Comment:
I think it's safer to user </create-project-workspaces> to close elements (when html is minified)